src/pages/Route/components/CreateStep4/CreateStep4.tsx

```diff
@@ -17,15 +18,18 @@ const style = {
 };
 
 const CreateStep4: React.FC<Props> = ({ form1, form2, redirect, ...rest }) => {
+
+  const { formatMessage } = useIntl();
+
   return (
     <>
-      <h2>定义 API 请求</h2>
+      <h2>{formatMessage({ id: 'route.create.define.api.request' })}</h2>
```

Comment:
Do not need the long key, we could use some words here, e.g route.create.defineAPIRequest